### ISSUE TITLE
Too slow

### ISSUE THREAD

**Issue by i00** [jhonhenry10/NPS_Browser#74]
LoadAllDatabaes is too slow, on occasion the servers are slow ... eg nopay ones ... these should download in parallel rather than one after the other..

Would be nice to have an offline-synced database too for speed.

**Comment by jhonhenry10** [comment]
for now, you can download all TSVs and direct app to local files
